doc/Hammer-Flow/Formal.rst

```diff
@@ -0,0 +1,67 @@
+Formal Verification
+===============================
+
+Hammer supports post-synthesis and post-P&R formal verification. It provides a simple API to provide a set of reference and implementation inputs (e.g. Verilog netlists) to perform formal verification checks such as logical equivalence checking (LEC).
+This action requires a tool plugin to implement ``HammerFormalTool``.
+
+Formal Verification Setup Keys
+-------------------------------
+
+* Namespace: ``vlsi.core``
+
+    * ``formal_tool_path``
+        * Set to the directory containing the tool plugin directory for the formal tool, typically ``/path/to/tool_plugin/formal``. This will be the parent directory of the directory containing ``__init__.py`` and ``defaults.yml``.
+    * ``formal_tool``
+        * Actual name of the formal verification tool that is setup in the directory ``formal_tool_path``, e.g. ``conformal``
+
+Formal Verification Input Keys
+-------------------------------
+
+* Namespace: ``formal.inputs``
+
+    * ``check`` (str)
+
+        * Name of the formal verification check that is to be performed. Support varies based on the specific tool plugin. Potential check types/algorithms could be "lec", "power", "constraint", "cdc", "property", "eco", and more.
```

Comment:
Similar to defaults.yml, can we make it a bit clearer what is currently supported?